deployment/helm/synopsys-alert/templates/alert-environ-configmap.yaml

```diff
@@ -1,13 +1,20 @@
 apiVersion: v1
 kind: ConfigMap
 data:
+  {{- if .Values.alert.hostname }}
+  ALERT_HOSTNAME: {{ .Values.alert.hostname | quote }}
+  {{- else }}
+  ALERT_HOSTNAME: {{ .Release.Name | quote }}-alert
```

Comment:
I don't think we need to put "-alert"

This PR removed the "-alert" from other areas of the Chart: https://github.com/blackducksoftware/blackduck-alert/pull/941